README.md

```diff
@@ -35,6 +29,32 @@ toc: true
 ---
 ```
 
+There are three Liquid filters available now.
+
+1. `toc_only`  
+   Generates the TOC itself as described [below](#generated-table-of-contents-html). 
+
+2. `inject_anchors`  
```

Comment:
I understood `toc_only` filter is used like `{{ content | toc_only }}`, but how do you use this `{{ inject_anchors }}`?

like {{ content | inject_anchors }}? and what is the result?